pages/hydra/developer_resources/groups.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Manage Groups of Users"
+keywords: Roles, Groups
+categories: How to Do All the Things - Glossary
+permalink: groups.html
+folder: hydra/how-to/
+sidebar: home_sidebar
+tags: [development_resources]
+toc: false
+---
+
+### The Default Setup
+
+Hydra ships with a YAML based group management system.  Typically this is only used when learning Hydra and you might replace it with something such as [Role Management](https://github.com/projecthydra/hydra-role-management) or [Rolify](https://github.com/RolifyCommunity/rolify), or [Hydra-ldap]( https://github.com/projecthydra-labs/hydra-ldap).
```

Comment:
How about put the YAML one first as this is the default behavior.  Then, do a lesson on "Database backed groups" using hydra-role-management.  That lesson be the existing admin-users.md page.  Both lessons could have the learning goal of "creating an admin group"